src/main/java/com/darkona/adventurebackpack/util/HungerOverhaulUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.darkona.adventurebackpack.util;
+
+import java.lang.reflect.Field;
+
+import com.darkona.adventurebackpack.reference.LoadedMods;
+
+public class HungerOverhaulUtils {
+
+    private HungerOverhaulUtils() {}
+
+    private static int milkedTimeout;
+
+    static {
+        if (LoadedMods.HUNGEROVERHAUL) {
+            readMilkedTimeout();
+        }
+    }
+
+    private static void readMilkedTimeout() {
+        if (Utils.inClient()) {
+            try {
+                Class<?> iguanaConfigClass = Class.forName("iguanaman.hungeroverhaul.config.IguanaConfig");
+                Field milkedTimeoutField = iguanaConfigClass.getDeclaredField("milkedTimeout");
+                milkedTimeout = milkedTimeoutField.getInt(null);
```

Comment:
If you decided to use reflection, please find necessary field *once* and save. And then reuse already saved field.

But you can access this field directly, since one is public.
To accomplish that, add HungerOverhaul as *optional* dependency (by adding next line in `dependencies.gradle`):
```
compileOnly('curse.maven:hunger-overhaul-224476:2332505') { transitive = false }
```

Reply:
I added it as dependency at the first time, however I found this mod uses reflection to access every other mods so I just followed that

see:
https://github.com/GTNewHorizons/AdventureBackpack2/blob/master/src/main/java/com/darkona/adventurebackpack/util/GregtechUtils.java
https://github.com/GTNewHorizons/AdventureBackpack2/blob/master/src/main/java/com/darkona/adventurebackpack/util/ThaumcraftUtils.java